akka-samples/akka-sample-cluster-java/build.sbt

```diff
@@ -5,44 +5,44 @@ val akkaVersion = "2.4-SNAPSHOT"
 
 val project = Project(
   id = "akka-sample-cluster-java",
-  base = file("."),
-  settings = Project.defaultSettings ++ SbtMultiJvm.multiJvmSettings ++ Seq(
-    name := "akka-sample-cluster-java",
-    version := "2.4-SNAPSHOT",
-    scalaVersion := "2.11.7",
-    scalacOptions in Compile ++= Seq("-encoding", "UTF-8", "-target:jvm-1.8", "-deprecation", "-feature", "-unchecked", "-Xlog-reflective-calls", "-Xlint"),
-    javacOptions in Compile ++= Seq("-source", "1.8", "-target", "1.8", "-Xlint:unchecked", "-Xlint:deprecation"),
-    javacOptions in doc in Compile := Seq("-source", "1.8", "-Xdoclint:none"),
-    libraryDependencies ++= Seq(
-      "com.typesafe.akka" %% "akka-actor" % akkaVersion,
-      "com.typesafe.akka" %% "akka-remote" % akkaVersion,
-      "com.typesafe.akka" %% "akka-cluster" % akkaVersion,
-      "com.typesafe.akka" %% "akka-cluster-metrics" % akkaVersion,
-      "com.typesafe.akka" %% "akka-cluster-tools" % akkaVersion,
-      "com.typesafe.akka" %% "akka-multi-node-testkit" % akkaVersion,
-      "org.scalatest" %% "scalatest" % "2.2.1" % "test",
-      "io.kamon" % "sigar-loader" % "1.6.6-rev002"),
-    javaOptions in run ++= Seq(
-      "-Xms128m", "-Xmx1024m", "-Djava.library.path=./target/native"),
-    Keys.fork in run := true,
-    mainClass in (Compile, run) := Some("sample.cluster.simple.SimpleClusterApp"),
-    // make sure that MultiJvm test are compiled by the default test compilation
-    compile in MultiJvm <<= (compile in MultiJvm) triggeredBy (compile in Test),
-    // disable parallel tests
-    parallelExecution in Test := false,
-    // make sure that MultiJvm tests are executed by the default test target,
-    // and combine the results from ordinary test and multi-jvm tests
-    executeTests in Test <<= (executeTests in Test, executeTests in MultiJvm) map {
-      case (testResults, multiNodeResults)  =>
-        val overall =
-          if (testResults.overall.id < multiNodeResults.overall.id)
-            multiNodeResults.overall
-          else
-            testResults.overall
-        Tests.Output(overall,
-          testResults.events ++ multiNodeResults.events,
-          testResults.summaries ++ multiNodeResults.summaries)
-    },
-    licenses := Seq(("CC0", url("http://creativecommons.org/publicdomain/zero/1.0")))
-  )
-) configs (MultiJvm)
+  base = file(".")
+).settings(Defaults.coreDefaultSettings ++ SbtMultiJvm.multiJvmSettings ++ Seq(
```

Comment:
Adding `Defaults.coreDefaultSettings` manually is not required as it is added by the `sbt.plugins.CorePlugin` AutoPlugin which is enabled by default on all projects.

I think this change should be reformatted like so:
```
val project = Project(
    id = "akka-sample-cluster-java",
    base = file(".")
  )
  .settings(SbtMultiJvm.multiJvmSettings: _*)
  .settings(
    name := "akka-sample-cluster-java",
    ...
  )
  .configs(MultiJvm)
```

Here I removed `Defaults.coreDefaultSettings`  and also used `settings` method override which accepts var args. This also has a nice little advantage that it does not change setting identation. :)